lib/SILOptimizer/Utils/ValueLifetime.cpp

```diff
@@ -55,30 +69,31 @@ void ValueLifetimeAnalysis::propagateLiveness() {
     if (bb == defBB && !hasUsersBeforeDef)
       continue;
 
-    for (SILBasicBlock *Pred : bb->getPredecessorBlocks()) {
+    for (auto *predBB : bb->getPredecessorBlocks()) {
       // If it's already in the set, then we've already queued and/or
       // processed the predecessors.
-      if (liveBlocks.insert(Pred))
-        worklist.push_back(Pred);
+      if (liveBlocks.insert(predBB))
+        worklist.push_back(predBB);
     }
   }
 }
 
 SILInstruction *ValueLifetimeAnalysis::findLastUserInBlock(SILBasicBlock *bb) {
   // Walk backwards in bb looking for last use of the value.
-  for (auto ii = bb->rbegin(); ii != bb->rend(); ++ii) {
-    assert(defValue != &*ii && "Found def before finding use!");
+  for (auto &ii : llvm::reverse(*bb)) {
```

Comment:
This confused me for a bit. I expected a variable called `ii` to be an instruction iterator. It's actually a reference to an instruction. Shouldn't it be called `inst` or `instruction`?

Reply:
Fixed.